port_exposer/src/main.rs

```diff
@@ -1,8 +1,8 @@
+use crate::port_exposer::PortExposer;
```

Comment:
I don't understand this move at all. 

Please move it back to the place where it was. At least somewhere under the copyright clause.

Reply:
This didn't move from anywhere, there was an error failed to resolve: use of undeclared type `PortExposer`. I imported and it put it there. I have moved it below the copyright clause.